src/plugin.js

```diff
@@ -740,6 +740,45 @@ const contribAdsPlugin = function(options) {
 
   };
 
+  // A utility method for textTrackChangeHandler to define the conditions
+  // when text tracks should be disabled.
+  // Currently this includes:
+  //  - on iOS with native text tracks, during an ad playing
+  const shouldDisableTracks = function() {
+    // If the platform matches iOS with native text tracks
+    // and this occurs during ad playback, we should disable tracks again.
+    // If shouldPlayContentBehindAd, no special handling is needed.
+    return !player.ads.shouldPlayContentBehindAd(player) &&
```

Comment:
Would appreciate a close look here to make sure I did the negation correctly!

Reply:
Looks right to me. Not live & ad playing & native text tracks & ios & not older vjs native text tracks